apisix/admin/consumers.lua

```diff
@@ -62,6 +63,12 @@ local function check_conf(username, conf)
         end
     end
 
+    if conf.group_id then
+        if consumer_group.get(conf.group_id) == nil then
```

Comment:
For code style, I will recommend using the same solution in https://github.com/apache/apisix/blob/3c18031c6e70daa15db72fe06ae8644b04b4481e/apisix/admin/routes.lua#L79
1. fetch the data from etcd if possible
2. use a similar error message